__tests__/api.test.js

```diff
@@ -149,7 +149,60 @@ describe("challenge 5, get/api/articles", () => {
     });
 
 
+  })
+});
+
+describe("challenge 6", () => {
+  test(`does the endpoint /api/:articleid/comments send back an array of comment objects with selected article id with correct keys?`, () => {
+    return request(app)
+      .get(`/api/1/comments`)
+      .expect(200)
+      .then(({body}) => {
+        body.commentsOnArticles.forEach((article) => {
+        expect(article).toMatchObject({
+          comment_id: expect.any(Number),
+          body: expect.any(String),
+          article_id: expect.any(Number),
+          author: expect.any(String),
+          votes: expect.any(Number),
+          created_at: expect.any(String)
+        })
+      })
+    })
+  })
+
+  test("Does the article id in the endpoint match the one in the returning array of objects?", () => {
+    return request(app)
+      .get(`/api/1/comments`)
+      .expect(200)
+      .then(({body}) => {
+        body.commentsOnArticles.forEach((article) => {
+        expect(article.article_id).toEqual(1)
+      })
+    })
+  })
```

Comment:
Ah I see you've made the check here, that's good 
You could eliminate the need for this test by being more specific about the article id in the test above though